ext/opcache/zend_accelerator_module.c

```diff
@@ -852,6 +852,11 @@ static ZEND_FUNCTION(opcache_compile_file)
 		return;
 	}
 
+	if (!accel_startup_ok) {
+		zend_error(E_NOTICE, ACCELERATOR_PRODUCT_NAME " has not been properly started, can't compile file");
```

Comment:
Unrelated: Is there even a need for the `ACCELERATOR_PRODUCT_NAME` to still be a thing as its development is managed by us?

Reply:
Frankly, I don't know, but in doubt we should stick with it.

Reply:
@dstogov Any input here? Is this something Zend uses internally still?

Reply:
could this error message contain a hint where to look for the problem, e.g. ini-setting `opcache.enable_cli`

Reply:
From what I can tell, OPcache might not be started for completely different reasons as well (e.g. because it failed to attach to the SHM on Windows).